java/offline-geocode/src/main/java/com/esri/arcgisruntime/sample/offlinegeocode/MainActivity.java

```diff
@@ -209,7 +209,7 @@ public void run() {
 
     // add a graphics overlay
     graphicsOverlay = new GraphicsOverlay();
-    graphicsOverlay.setSelectionColor(Color.CYAN);
+    //graphicsOverlay.setSelectionColor(Color.CYAN);
```

Comment:
You deleted this lines on the other samples, i don't know if you want to keep them comment out